src/codeu/chat/util/PlayInfo.java

```diff
@@ -1,22 +1,36 @@
 package codeu.chat.util;
```

Comment:
General advice: Util classes shouldn't hold any data relevant to the application. These should be static functions, like black boxed, that take some input, do some processing, and provide an output that is then used by the server or client. So in this case, although load50Line() and loadRoles() are functions that should be in a util class, they should return these lines (and other data) to some class that sits on the server (or maybe chat/common).